assets/sass/components/_frontpage-cards.scss

```diff
@@ -17,6 +19,10 @@
 				font-size: 18px;
 				margin-top: 20px;
 				margin-bottom: 10px;
+
+				a {
+					@include focus-with-background-and-color;
```

Comment:
And if these are merged, some guidelines for the usage would be welcome :) so, when to use which one.

Reply:
Yes, agreed.

I didn't do anything here, as I thought that's a job for the design system.

Reply:
For that, I think you can write documentation in the SCSS file. Or in the style guides repo (assuming these are going there some day).